### PR TITLE
Adds changelog parameter and fancy version parameters into all-citus

### PR DESCRIPTION
**Pull request by gurkanindibay**
Added a new changelog parameter to manually add changelogs

Depends on https://github.com/citusdata/tools/pull/183

### CODE REVIEW

File: .github/workflows/update_package_properties.yml
Comment (hanefi) [inline .github/workflows/update_package_properties.yml]: needs to be updated

Comment (hanefi) [reply .github/workflows/update_package_properties.yml]: @gurkanindibay Please update the PR description to indicate that this PR is blocked on a tools PR.

If you do not provide a link, it requires digging everytime. I do not want to spend time going through tools repo PRs and see which one actually has `update_properties_fixes` as the base branch.

File: .github/workflows/update_package_properties.yml
Comment (hanefi) [inline .github/workflows/update_package_properties.yml:26]: You are missing `required` properties here. All inputs entries should have one.

File: .github/workflows/update_package_properties.yml
Comment (gurkanindibay) [reply .github/workflows/update_package_properties.yml]: I've already added one in description https://github.com/citusdata/packaging/pull/782#issue-737431653

File: .github/workflows/update_package_properties.yml
Comment (gurkanindibay) [reply .github/workflows/update_package_properties.yml:26]: Fixed

File: .github/workflows/update_package_properties.yml
Comment (hanefi) [reply .github/workflows/update_package_properties.yml]: > I've already added one in description [#782 (comment)](https://github.com/citusdata/packaging/pull/782#issue-737431653)

No, I did that. Please provide links in the PR description in the future.

**Review by hanefi — DISMISSED** [review]
Don't forget to update the branch name after citusdata/tools#183 is merged

**Review by hanefi — APPROVED** [review]

**Pull request merged**
